packages/theme/src/plugin.ts

```diff
@@ -100,7 +100,33 @@ export const mochiui = () => {
   return plugin(
     ({ addVariant, addUtilities }) => {
       // add the css variables to "@layer utilities"
-      addUtilities({ ...resolved?.utilities })
+      addUtilities({
+        ...resolved?.utilities,
+        '.landing-container': {
+          width: '100%',
+          maxWidth: '1280px',
+          marginLeft: 'auto',
+          marginRight: 'auto',
+          paddingLeft: '16px',
+          paddingRight: '16px',
```

Comment:
Please using rem unit
```suggestion
          paddingLeft: '1rem',
          paddingRight: '1rem',
```